example_config/example_config_base.yaml

```diff
@@ -177,6 +177,17 @@ ses:
 challenge_url:
   enabled: true
 
+# This section specifies which roles consoleme is allowed to pull, cache & display. There may still be workarounds via
+# cURL to create requests directly against consoleme. If you have to block cURL requests for requests against specific
+# roles, it's probably best to enforce IAM permissions for consoleme to only modify specific roles. You should only
+# need one or the other parameter defined below
+# Parameters:
+#  allowed_tags: map[string]string: if non-empty, consoleme will only consider roles tags mapped here
+#  allowed_arns: list[string]:  if non-empty, consoleme will only consider the role arns in this list
+roles:
```

Comment:
You can leave these 3 lines also commented out, since most people can uncomment if they need them!